diff --git a/backends/smt2/smtio.py b/backends/smt2/smtio.py
index 0ec7f08f4..9984bba97 100644
--- a/backends/smt2/smtio.py
+++ b/backends/smt2/smtio.py
@@ -210,10 +210,7 @@ class SmtIo:
                 sys.exit(1)
 
         if self.solver in ["boolector", "bitwuzla"]:
-            if self.noincr:
-                self.popen_vargs = [self.solver, '--smt2'] + self.solver_opts
-            else:
-                self.popen_vargs = [self.solver, '--smt2', '-i'] + self.solver_opts
+            self.popen_vargs = [self.solver, '--lang', 'smt2'] + self.solver_opts
             self.unroll = True
             if self.timeout != 0:
                 print('timeout option is not supported for %s.' % self.solver)
